### PR TITLE
AT_01.01_002 | Verify Jenkins icon redirection

### PR DESCRIPTION
**Pull request by ellayezina**
https://trello.com/c/f2eaBNP0/122-at0101002-verify-jenkins-icon-redirection

### CODE REVIEW

File: cypress/e2e/headIcon.cy.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+describe('Jenkins icon', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:8080/')
+    })
```
Comment (Rom-T) [inline cypress/e2e/headIcon.cy.js:5]: We don't write visit because we are already on the home page

File: cypress/e2e/headIcon.cy.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+describe('Jenkins icon', () => {
```
Comment (Rom-T) [inline cypress/e2e/headIcon.cy.js:2]: Please rename describe to 'Head Icon'.

**Review by ArtyomDulya — APPROVED** [review]

File: cypress/e2e/headIcon.cy.js (hunk 1 above)
Comment (ellayezina) [reply cypress/e2e/headIcon.cy.js]: Fixed

File: cypress/e2e/headIcon.cy.js (hunk 2 above)
Comment (ellayezina) [reply cypress/e2e/headIcon.cy.js]: Fixed

**Review by lanasussman — APPROVED** [review]
Well done!

**Review by Rom-T — APPROVED** [review]

**Pull request merged**